src/tools/pub/custom-package-repositories.md

```diff
@@ -220,6 +220,10 @@ your own custom package repository:
   <img src="/assets/img/tools/jfrog.svg" width="48" alt="JFrog logo">
   <a href="https://www.jfrog.com/confluence/display/JFROG/Pub+Repositories"><b>JFrog Artifactory</b></a>
 </li>
+<li>
+  <img src="/assets/img/tools/onepub.svg" width="48" alt="OnePub logo">
+  <a href="https://onepub.dev"><b>OnePub Dart Repository</b></a>
```

Comment:
One question. Is "Dart Repository" part of the name or is it simply "OnePub"? 

If it's just "OnePub", "Dart Repository" may be unnecessary since its apparent from the context, similar to Cloudsmith's entry.